packs/packer/actions/push.py

```diff
@@ -0,0 +1,15 @@
+from lib.actions import BaseAction
+
+
+class PushAction(BaseAction):
+    def run(self, packerfile, name, message=None, cwd=None, exclude=None,
+            only=None, variables=None, variables_file=None):
+        if cwd:
+            self.set_dir(cwd)
```

Comment:
Packer is a little fickle. It expects certain folders referenced in the definitions relative to where the script lives. To that end, you have to include this along with the definition itself.